logos.json

```
@@ -1341,6 +1341,21 @@
 		"category": ["EdTech"]
 	},
 	{
+		"title": "Peerstack",
+		"filename": "peerstack",
+		"url": "https://www.thepeerstack.com",
+		"category": [
+			"Financial Services",
+			"Software"
+		]
+	},
+	{
+		"title": "Serah Kassim",
+		"filename": "serah_kassim",
+		"url": "https://serahkassim.com/",
+		"category": ["fashion"]
```

```suggestion
		"category": ["Fashion"]
```

Hi @ichtrojan sorry this fell off my radar. Kindly make this one last change so I can merge and take it live